File: py-c/working on assignments/conditionals/oldEnough.c
#include <stdio.h>
int age;
int main(void){
    printf("what is your age: ");
    scanf("%d",&age);
    if (age >= 18){
        printf("you can vote!\n");
    }else if (age >= 16){
        printf("you can drive!\n");
    }else if (age >= 15){
        printf("you can get a learners permit!\n");
    }else if (age >= 5){
        printf("you can go to school!\n");
    }else{
        printf("you cant do anything\n");
    }
    return 0;
}

File: py-c/working on assignments/conditionals/oldEnough.py
age = int(input("what is your age: "))

if age >= 18:
    print("you are old enough to vote!")
elif age >= 16:
    print("you are old enough to drive!")
elif age >= 15:
    print("you are old enough to get a learners permit!")
elif age >= 5:
    print("you are old enough to go to school!")
else:
    print("you cant do anything:(")